web/src/mobile/components/Orders/OrdersItem.tsx

```diff
@@ -41,29 +42,61 @@ const OrdersItemComponent = props => {
             </div>
             <div className="pg-mobile-orders-item__row">
                 <div className="pg-mobile-orders-item__row__block">
-                    <span>{intl.formatMessage({ id: 'page.mobile.orders.header.amount' })}</span>
-                    <span className="pg-mobile-orders-item__row__block__value">
-                        <Decimal fixed={currentMarket.amount_precision}>{order.origin_volume}</Decimal>
-                    </span>
+                    <span>{intl.formatMessage({ id: 'page.mobile.orders.header.filled' })}</span>
+                    <div className="pg-mobile-orders-item__row__block__value">
+                        <span style={{
+                            color:  setTradeColor(order.side).color
+                        }}>
+                            <Decimal
+                              fixed={FIXED_VOL_PRECISION}
+                              color={setTradeColor(order.side).color}
+                            >
+                                {parseFloat(filled)}
```

Comment:
Decimal is the component that we use instead of parse float. Remove please parseFloat